pipelines/dump/Dockerfile

```
@@ -0,0 +1,10 @@
+ARG NODE_BASEIMAGE=docker.dbc.dk/dbc-jessie:latest
```

Tænker ikke det giver mening at kalde variablen node_baseimage
```suggestion
ARG BASEIMAGE=docker.dbc.dk/dbc-jessie:latest
```

---

en klassisk copy/paste situation